modules/lib/src/main/resources/assets/js/app/settings/wizard/panel/form/element/ProjectAccessControlEntryView.ts

```diff
@@ -2,17 +2,25 @@ import {ProjectAccessControlEntry} from '../../../../access/ProjectAccessControl
 import {Principal} from '@enonic/lib-admin-ui/security/Principal';
 import {ProjectAccessSelector} from './ProjectAccessSelector';
 import {PrincipalContainerSelectedEntryView} from '@enonic/lib-admin-ui/ui/security/PrincipalContainerSelectedEntryView';
+import {PrincipalContainerSelectedOptionView} from '@enonic/lib-admin-ui/ui/security/PrincipalContainerSelectedOptionView';
 import {ProjectAccessValueChangedEvent} from '../../../../event/ProjectAccessValueChangedEvent';
+import {ProjectACESelectedOptionsView} from './ProjectAccessControlComboBox';
+
+type projectACESelectedOptionsViewChild = PrincipalContainerSelectedOptionView<ProjectAccessControlEntry>;
 
 export class ProjectAccessControlEntryView
     extends PrincipalContainerSelectedEntryView<ProjectAccessControlEntry> {
 
+    private projectACESelectedOptionsView: ProjectACESelectedOptionsView;
```

Comment:
I think this is bad idea to pass and keep a reference to it's parent: in general class shouldn't know nothing about it's structure and it's parents